packages/types/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@slack/types",
-  "version": "2.16.0",
+  "version": "2.16.0-aiAppsBeta.3",
```

suggestion: We'll need to revert this version
```suggestion
  "version": "2.16.0",
```

---

@mwbrooks We've undone such in https://github.com/slackapi/node-slack-sdk/pull/2399/commits/b8a3ab02052671d728d88515e43a925bf8bf5c7a.